AdventaR/app/components/arScene.js

```diff
@@ -71,7 +75,7 @@ export default class HelloWorldSceneAR extends Component {
   };
 
   componentDidMount(){
-    ReactNativeHeading.start(15)
+    ReactNativeHeading.start(1)
```

Comment:
We might long-term need a better solution to this, if we have time.  This seems like it might eat battery.